README.md

```diff
@@ -1,2 +1,841 @@
-# itn-did-spec
-ITN DID Method Specification
+The DID ITN Method Specification 1.0
+==================
+
+**Specification Status**: Working Draft
+
+**Latest Draft:** [ITN DID Specification](https://github.com/itn-trust/itn-did-spec)
+
+**Draft Created:** April 10, 2024
+
+**Latest Update:** April 11, 2024
+
+**Contributors & Editors:**
+
+1. [Andreas Freund](https://github.com/Therecanbeonlyone1969)
+2. [Dmitry Zotov](https://github.com/AlexMesser)
+3. [Parth Bhatt](https://github.com/bparth24)
+4. [Umed Khudoiberdiev](https://github.com/pleerock)
+
+
+## Abstract
+
+Integrated Trust Network (ITN) is a community-built and operated network. The objective is to build a Federated Trust Layer (Core Services) and Business Automation Interoperability Infrastructure (User Agents) for ITN. The ITN Identifier is specified in accordance with the [W3C DID Core specification](https://www.w3.org/TR/did-core/).
+
+ITN Identifier = ITN DID
+
+The syntax of the identifier and accompanying data model used by the protocol is conformant with the [DID-CORE](https://www.w3.org/TR/did-core/) specification and shall be registered with the [DID-SPEC-REGISTRIES](https://www.w3.org/TR/did-spec-registries/).
+
+For a detailed read on DID and other DID method specifications, one can refer [WebOfTrustInfo - DID Primer](https://github.com/WebOfTrustInfo/rwot5-boston/blob/master/topics-and-advance-readings/did-primer.md)
+
+## Architecture Overview
```

Comment:
@bparth24 we need an explanation on how did:itn works from an architectural point of view. That is most critical and totally missing. we have that stuff canned already. so it is cut-and-paste. Agent architecture, CAS, multi-chains etc.

Reply:
We need an example upfront what a typical ITN DID doc looks like.

We also need to include what a resolved did doc looks like including meta-data since did resolution is part of the spec and not here afais.

Reply:
Completely agree with @Therecanbeonlyone1969 .

This document must cover high level of what did:itn is from architectural point of view. More like a theory. We don't need all those "SDK" and "APIs". I would say I would start with zero implementation details, only abstractions and concepts. Other specifications I see don't have implementation details.